### DISCUSSION TITLE
Set spacing between list items

### DISCUSSION THREAD

**Discussion by shawn-frank** [gonzalezreal/swift-markdown-ui#320] [Q&A]
Is it possible to set custom spacing between list items in a list ?

I have some basic markdown like this:

- This is item 1
- This is item 2

Is it possible to set the spacing between the items in a list which is seperate from the paragraph spacing that we set ?

**Comment by gonzalezreal** [comment]
This might help:
https://gonzalezreal.github.io/2023/02/18/better-markdown-rendering-in-swiftui.html#styling-blocks

Some useful excerpts:

> MarkdownUI ignores the bottom margin preference when rendering tight lists. So, you should consider using the top margin preference to make adjustments.

> For loose lists, MarkdownUI considers the bottom margin. In the snippet above, as we didn’t define one for the list item, it takes the bottom margin preference of the paragraph style, as list items contain paragraphs.
